app/constants/currencies.js

```diff
@@ -0,0 +1,35 @@
+export const currencies = [
```

Comment:
You can't do this because how we can know an address is about Ethereum, Polygon...